src/Tribe/Generator/Event.php

```diff
@@ -465,5 +466,6 @@ public function add_ticket( $event ) {
 		];
 
 		$provider->ticket_add( $event->ID, $data );
+        add_post_meta( $event->ID, '_EventCost', $price );
```

Comment:
```suggestion
        update_post_meta( $event->ID, '_EventCost', $price );
```I'd use update - as a just-in-case.